src/main/java/Ui.java

```diff
@@ -0,0 +1,146 @@
+import java.util.ArrayList;
+
+/**
+ * Ui class deals with interactions with the user.
+ */
+public class Ui {
+    private static final String LINE = "____________________________________________________________" + "\n";
```

Comment:
Perhaps the HORIZONTAL_LINE under duke.java can be removed since this is here? :D